KonyAccelerate/controllers/mobile/frmLoginController/frmLoginController.js

```diff
@@ -1,6 +1,74 @@
-define({ 
+define({
 
- //Type your controller code here 
- 
+    /**
+     *	@function fetchEventData
+     * 	@description This function is used to fetch all the event data required
+     * 	@private
+     */
+    fetchEventData: function() {
+        var objectServiceName = eventConstants.OBJECT_SERVICE_NAME;
+        var objectName = eventConstants.EVENT_OBJECT_NAME;
+        var queryParams = {
+            "$filter": "((SoftDeleteFlag ne true) or (SoftDeleteFlag eq null))",
+            "$expand": "event_inner_location,wifi_info,location"
```

Comment:
if possible try to move these values to  event_inner_location , wifi_info, location to constants file